src/lib/forms/AutogenerateForm.svelte

```diff
@@ -51,7 +51,7 @@
 </script>
 
 {#if spec.description}
-  <p>{spec.description}</p>
+  <p class="govuk-body">{spec.description}</p>
```

Comment:
I bet we could make every `<p>` do this centrally by using [global](https://svelte.dev/docs/svelte-components#style). Only in some places we may want a different paragraph style

Reply:
I don't think we can do this without copying the stylings manually, which I don't think would be good practice. I think we can use global to apply styling to e.g. each `<p>`, but we I don't think we can use it to apply classes to each `<p>`

Reply:
I have reduced number of `class="govuk-body"`, though by simply applying it something higher in the DOM hierarchy. I assume we basically want everythign to have the gov uk font so it seems easiest and valid

Reply:
I think we should find a way to avoid having to write `class` for all/most elements in our code. Two reasons:

1) It's easy to miss one spot, or forget to add it to something new
2) It adds visual noise and violates DRY

Ideally there'd be a way to tell CSS "select all p elements and apply the govuk-body class to them." I haven't found a way to do that yet...

A backup plan is to define our own `<P>`, `<Button>`, etc Svelte components, like https://github.com/acteng/atip/blob/main/src/lib/common/ExternalLink.svelte. It'd be annoying to have to import those in a bunch of places, but is better IMO than writing `class` everywhere.

There's a Slack for asking questions, right? Or we could search / ask on https://github.com/alphagov/govuk-frontend about common practices here.

Reply:
Hmm interesting. Sounds like a nice solution for button for sure. I think writing `class="govuk-body"` or similar at the top level of most components might not be too bad... But again, maybe we just want everything inside a `GovUkBody.svelte` component.